obspy/io/seiscomp/tests/data/iris_events.sc3ml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<seiscomp xmlns="http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.9">
+<seiscomp xmlns="http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.9" version="0.9">
   <EventParameters publicID="smi:www.iris.edu/ws/event/query">
     <origin publicID="smi:www.iris.edu/ws/event/query?originId=7680412">
       <time>
-        <value>2011-03-11T05:46:24.120000Z</value>
+        <value>2011-03-11T05:46:24.1200</value>
```

Comment:
The write module was tested by reading QuakeML files and writing SC3ML files using ObsPy [read_events](https://docs.obspy.org/master/packages/autogen/obspy.core.event.read_events.html) method. Now, the unit tests don't use the [read_events](https://docs.obspy.org/master/packages/autogen/obspy.core.event.read_events.html) method, the QuakeML file is directly converted in SC3ML with the XSLT file. I think it's more accurate since what we really want to test is the XSLT file. However, the result isn't exactly the same. The date format of the original file isn't modified when using the XSLT file.

By the way, sorry for the late answer!